.github/workflow-templates/bot-coin-family.yml

```diff
@@ -38,46 +38,15 @@ jobs:
       - name: Retrieving coin apps
```

Comment:
is it not possible to put the coin apps on the action ?

Reply:
i'm not sure it's a good idea.

a given new coin may want to use "master" instead of "ledger-live-bot" maybe because their coin is not there yet.

Reply:
oh also it's counterintuitive, but it's a better idea to minimize the amount of steps in a "composite" because it gets packages into one run and you don't have the details anymore on the CI (if you see, the bot part is one step, even tho it contains the build part in it)